contracts/tgrade-valset/Cargo.toml

```diff
@@ -48,3 +48,8 @@ tg-bindings-test = { path = "../../packages/bindings-test", version = "0.6.0-alp
 derivative = "2"
 anyhow = "1"
 assert_matches = "1.5"
+
+# For integration tests
+cosmwasm-std = { version = "1.0.0-beta4", default_features = false }
```

Comment:
Can we make these all optional?
Only included with "integration" feature.

(I have a Mac m1 and won't be able to build/test this contract if these are present)